cmd/nfd-worker/main.go

```diff
@@ -34,21 +49,14 @@ import (
 )
 
 const (
-	// ProgramName is the canonical name of this discovery program.
-	ProgramName = "node-feature-discovery"
-
-	// Namespace is the prefix for all published labels.
-	labelNs = "feature.node.kubernetes.io/"
-
-	// Namespace is the prefix for all published labels.
-	annotationNs = "nfd.node.kubernetes.io/"
+	// ProgramName is the canonical name of this program
+	ProgramName = "nfd-slave"
```

Comment:
```suggestion
	ProgramName = "nfd-worker"
```
This would be consistent with the executable name. 
```
nfd-worker -h
```
prints out nfd-slave

Reply:
Thanks for spotting this. I did some bikeshedding around the naming and this was a leftover, as you can guess...